### PR TITLE
Fixes discussion new/edit form

### PR DESCRIPTION
**Pull request by joaojusto**
Before: 
screen shot 2015-01-30 at 17 03 30
After: 
screen shot 2015-01-30 at 17 03 38

### CODE REVIEW

**Comment by joaojusto** [comment]
@gabrielpoca bump

File: app/assets/javascripts/app/discussions/templates/desktop.html.slim [hunk 1]
```diff
@@ -22,4 +23,4 @@ TopBarLayout
         label Tags
         input ng-model="ctrl.discussion.tags" type="text"
 
-      textarea.Editor placeholder="Write the fundaments of the discussion" ng-model="ctrl.discussion.body" required=""
+      textarea.Editor.Editor-noSidePadding placeholder="Write the fundaments of the discussion" ng-model="ctrl.discussion.body" required=""
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/templates/desktop.html.slim]: `noSidePadding` isn't a good name, and please read [this](https://github.com/suitcss/suit/blob/master/doc/README.md), you have been using it for months and still have no idea what you're doing

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/templates/desktop.html.slim]: What about Editor--noPadding?

Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/templates/desktop.html.slim]: That would be the right usage of SUIT. But the name is still bad, it should not be couple to the rules.

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/templates/desktop.html.slim]: .Editor--discussionForm?

**Comment by joaojusto** [comment]
@gabrielpoca

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**